JBD2: fix up thinko in jbd2 revoke code

From: James Bottomley <James.Bottomley@HansenPartnership.com>

This commit:

commit e898f3154d78d1520265aade0992981866d827fb
Author: Duane Griffin <duaneg@dghda.com>
Date:   Tue Mar 25 19:47:39 2008 -0400

    jbd2: tidy up revoke cache initialisation and destruction

Introduced a thinko (it doesn't check whether jbd2_revoke_record cache
is null, it checks jbd2_revoke_table_cache twice).

Signed-off-by: James Bottomley <James.Bottomley@HansenPartnership.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>

---

---
 fs/jbd2/revoke.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.25-rc9/fs/jbd2/revoke.c
===================================================================
--- linux-2.6.25-rc9.orig/fs/jbd2/revoke.c	2008-04-14 10:25:25.000000000 -0700
+++ linux-2.6.25-rc9/fs/jbd2/revoke.c	2008-04-14 10:25:26.000000000 -0700
@@ -192,7 +192,7 @@ int __init jbd2_journal_init_revoke_cach
 	jbd2_revoke_table_cache = kmem_cache_create("jbd2_revoke_table",
 					   sizeof(struct jbd2_revoke_table_s),
 					   0, SLAB_TEMPORARY, NULL);
-	if (jbd2_revoke_table_cache && jbd2_revoke_table_cache) {
+	if (jbd2_revoke_record_cache && jbd2_revoke_table_cache) {
 		return 0;
 	} else {
 		jbd2_journal_destroy_revoke_caches();
